### PR TITLE
feat: [sc-25979] Migrate NameGuard from serverless to Terraform deployment

### PR DESCRIPTION
**Pull request by BanaSeba**
Story details: https://app.shortcut.com/ps-web3/story/25979

### CODE REVIEW

File: apps/api.nameguard.io/package.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-    "NameKit",
-    "NameGuard",
-    "NameHash"
-  ],
-  "scripts": {
-    "deploy:prod": "DOCKER_CONFIG=.docker serverless deploy --stage prod",
-    "deploy:staging": "DOCKER_CONFIG=.docker serverless deploy --stage staging"
-  },
-  "dependencies": {
-    "serverless": "3.39.0"
```
Comment (lightwalker-eth) [inline apps/api.nameguard.io/package.json:24]: @BanaSeba suggest undoing the delete of this file. And then navigating to this directory and running `pnpm remove serverless`. We need PNPM to remove this dependency. The lock file in the root of the monorepo should be getting updated in this PR too.

File: apps/api.nameguard.io/package.json [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-  },
-  "devDependencies": {
-    "serverless-prune-plugin": "2.0.2"
```
Comment (lightwalker-eth) [inline apps/api.nameguard.io/package.json:27]: @BanaSeba Please see comment above. We also need to ask PNPM to remove this dependency too. We don't remove it just by deleting this file.

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@BanaSeba Hey great to see this. Reviewed and shared feedback :+1:

File: apps/api.nameguard.io/package.json (hunk 1 above)
Comment (BanaSeba) [reply apps/api.nameguard.io/package.json:24]: Updated pnpm as well

File: apps/api.nameguard.io/package.json (hunk 2 above)
Comment (BanaSeba) [reply apps/api.nameguard.io/package.json:27]: Updated pnpm as well

**Review by lightwalker-eth — APPROVED** [review]
@BanaSeba Excited to say goodbye to Serverless and these outdated dependencies 🚀 Thank you! 🙌

**Pull request merged**